xml/ntp_banners.xml

```diff
@@ -599,137 +599,137 @@ NTP "banners", taken from a readvar response
-   <param pos="0" name="os.version" value="2.4" />
-   <param pos="1" name="os.arch"/>
+    <param pos="0" name="os.vendor" value="Sun"/>
+    <param pos="0" name="os.family" value="Solaris"/>
+    <param pos="0" name="os.product" value="Solaris"/>
+    <param pos="0" name="os.version" value="2.4"/>
+    <param pos="1" name="os.arch"/>
   </fingerprint>
   <fingerprint pattern="^.*processor=&quot;([^ ]+)&quot;,.*system=&quot;SunOS/?5.5&quot;" flags="REG_DOT_NEWLINE,REG_ICASE">
-   <description>Solaris 2.5 (SunOS/5.5) with no ntp version</description>
-   <example os.arch="sun4m">
+    <description>Solaris 2.5 (SunOS/5.5) with no ntp version</description>
+    <example os.arch="sun4m">
      processor="sun4m", system="SunOS/5.5"
    </example>
-   <example os.arch="sun4u">
+    <example os.arch="sun4u">
     processor="sun4u", system="SunOS5.5"
    </example>
-   <param pos="0" name="os.vendor" value="Sun"/>
-   <param pos="0" name="os.family" value="Solaris"/>
-   <param pos="0" name="os.product" value="Solaris"/>
-   <param pos="0" name="os.version" value="2.5" />
-   <param pos="1" name="os.arch"/>
+    <param pos="0" name="os.vendor" value="Sun"/>
+    <param pos="0" name="os.family" value="Solaris"/>
+    <param pos="0" name="os.product" value="Solaris"/>
+    <param pos="0" name="os.version" value="2.5"/>
+    <param pos="1" name="os.arch"/>
   </fingerprint>
   <fingerprint pattern="^.*processor=&quot;([^ ]+)&quot;,.*system=&quot;SunOS/?5.6&quot;" flags="REG_DOT_NEWLINE,REG_ICASE">
-   <description>Solaris 2.6 (SunOS/5.6) with no ntp version</description>
-   <example os.arch="sun4m">
+    <description>Solaris 2.6 (SunOS/5.6) with no ntp version</description>
+    <example os.arch="sun4m">
      processor="sun4m", system="SunOS/5.6"
    </example>
-   <example os.arch="sun4u">
+    <example os.arch="sun4u">
     processor="sun4u", system="SunOS5.6"
    </example>
-   <param pos="0" name="os.vendor" value="Sun"/>
-   <param pos="0" name="os.family" value="Solaris"/>
-   <param pos="0" name="os.product" value="Solaris"/>
-   <param pos="0" name="os.version" value="2.6" />
-   <param pos="1" name="os.arch"/>
+    <param pos="0" name="os.vendor" value="Sun"/>
+    <param pos="0" name="os.family" value="Solaris"/>
+    <param pos="0" name="os.product" value="Solaris"/>
+    <param pos="0" name="os.version" value="2.6"/>
+    <param pos="1" name="os.arch"/>
   </fingerprint>
   <fingerprint pattern="^.*processor=&quot;([^ ]+)&quot;,.*system=&quot;SunOS/?5.([789]|10)&quot;" flags="REG_DOT_NEWLINE,REG_ICASE">
-   <description>Solaris 7-10 (SunOS/5.7 - SunOS/5.10) with no ntp version</description>
-   <example os.arch="sun4m" os.version="7">
+    <description>Solaris 7-10 (SunOS/5.7 - SunOS/5.10) with no ntp version</description>
+    <example os.arch="sun4m" os.version="7">
      processor="sun4m", system="SunOS/5.7"
    </example>
-   <example os.arch="sun4m" os.version="10">
+    <example os.arch="sun4m" os.version="10">
```

Comment:
Indention here is too deep vs closing tag.  Should it just be moved to a single line for opening/closing tags + content like fingerprint examples in other files?

Reply:
It seems like the regex is meant to work over multiple lines, so I think we should just keep it as is.  The whitespace matters here, but I don't know why xmllint isn't handling the indention as we'd like.  For now, I say we just leave it as is.